File: rasppi/mqtt.py
import paho.mqtt.client as mqtt

# Call back when client connects to the broker
def on_connect(client, userdata, flags, rc):
    print("Connected with result code "+str(rc))

# Create an MQTT client instance
client = mqtt.Client()

# Set the callback functions
client.on_connect = on_connect

# Connect to the broker
client.connect("mqtt-dashboard.com", 1883, 60)

# Publish a message
while(True):
    a = input("Enter your message: ")
    if a:
        client.publish("sim/kmutt/iot", a)

# keep the client running to process callbacks and messages
client.loop_forever()

File: server/mqtt.js
const mqtt = require('mqtt');

// MQTT client
const client = mqtt.connect('mqtt://mqtt-dashboard.com:1883');

// Callback when the client is connected
client.on('connect', () => {
    console.log('MQTT client connected');
    // Subscribe to the topic 'test'
    client.subscribe('sim/kmutt/iot');
});

// Callback when a message is received
client.on('message', (topic, message) => {
    console.log('Message received: ' + message.toString());
});